apps/docs/src/app/core/component-docs/switch/examples/switch-binding-example/switch-binding-example.component.scss

```diff
@@ -10,3 +10,7 @@ button {
 fd-switch {
     margin-bottom: 20px;
 }
+
+.switch-info {
```

Comment:
should be prefixed with `docs` so the user is aware this is a class for the documentation.

Reply:
I'd better apply inline style for the margin :)

Reply:
done

Reply:
👍